src/app/components/BankBoard.ts

```diff
@@ -20,11 +15,15 @@ export class BankBoard {
           animal.theName,
```

Comment:
The method in the line14 creates a new object, just to get the path. I asked @Enessetere to modify it in another file, perhaps you could look how he handled it or discuss with him. The path for animals are in the banks Herd, it is our source of information, we should retrieve it from there, not create new animal, which btw will be only present as Animal, Predator, Protector. Their path shoudl be retrieved from bank.theHerd.theAnimals and each object defining the animal.